### DISCUSSION TITLE
[Kavita] Prioritise Book 1 for Series Cover and Summary

### DISCUSSION THREAD

**Discussion by mslmn** [Kareadita/Kavita#3385] [Ideas]
### Idea Description

**Current Behavior:**
Kavita currently uses the first book in the series order for the series’ summary and cover. This works well for most series, as the first book in the order is typically "Book 1."

However, some series include novellas, prequels, or short stories with a series number less than 1. For instance, in _[The Green Bone Saga](https://hardcover.app/series/the-green-bone-saga)_, the novella numbered 0.5 (_The Jade Settler of Janloon_) is used by Kavita to populate the series’ summary and cover, instead of the primary starting point for the series, which is "Book 1" (_Jade City_).

This can result in less intuitive or unintended displays for users who expect the series' first main entry to define its summary and cover.

**Proposed Behavior:**
Provide an option for Kavita to always prioritise the first main entry (e.g., "Book 1") in a series as the default source for the summary and cover. This ensures that the series' primary starting point is highlighted in the library and aligns with user expectations for how a series should be represented.

### Idea Category

User Experience

### Duration of Using Kavita

2 years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
I think this was a bug in v0.8.3. Can you retest in v0.8.4? It is intended behaviour to prioritize the 1st (issue/book 1) book.

**Reply by mslmn** [reply]
The same thing is happening in v0.8.4 Docker. The series summary and cover aren't being populated from book 1. 

[The Green Bone Saga](https://www.goodreads.com/series/216281-the-green-bone-saga)
![kavita](https://github.com/user-attachments/assets/a5bc9d83-4a04-47ff-84ca-035c9cd23ea0)

[Raven's Shadow](https://www.goodreads.com/series/81339-raven-s-shadow)
![kavita2](https://github.com/user-attachments/assets/ce943ca0-da66-4baa-8c18-e4e422eb858f)

[His Dark Materials](https://www.goodreads.com/series/44427-his-dark-materials)
![kavita4](https://github.com/user-attachments/assets/c0acb238-9f16-43bb-ac26-c7827d1ea824)

[Powder Mage](https://www.goodreads.com/series/99602-powder-mage)
![kavita5](https://github.com/user-attachments/assets/fe4a675b-bb1f-4753-8e2d-15642ab7f2f4)

[Gods of Blood and Powder](https://www.goodreads.com/series/172832-gods-of-blood-and-powder)
![kavita3](https://github.com/user-attachments/assets/778d22eb-7bf7-47be-b8da-13f38bb09e8e)
